test/genegraph/rocksdb_test.clj

```diff
@@ -23,9 +23,18 @@
       (sut/rocks-put-multipart-key! db ["keyprefix1" "keysuffix1"] "value3")
       (sut/rocks-put-multipart-key! db ["keyprefix1" "keysuffix2"] "value4")
       (sut/rocks-put-multipart-key! db ["keyprefix2" "keysuffix2"] "value5")
+      (is (= 2 (-> (sut/prefix-seq db "keyprefix1") count)))
+      (is (= 1 (-> (sut/prefix-seq db "keyprefix2") count)))
```

Comment:
Why not just `(count (sut/prefix-seq db "keyprefix1"))`?
That way you see expected `count` right after the integer.